### PR TITLE
[ResourceBundle] class names with lower-case names

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    | no
| Deprecations? |no

### CODE REVIEW

**Comment by dpfaffenbauer** [comment]
@BlackbitDevs WDYT?

File: src/CoreShop/Bundle/ResourceBundle/Controller/ResourceSettingsController.php [hunk 1]
```diff
@@ -59,8 +59,13 @@ public function getConfigAction(): Response
                 $application = $alias[0];
                 $alias = $alias[1];
 
-                $class = str_replace('Pimcore\\Model\\DataObject\\', '', $definition['classes']['model']);
-                $class = str_replace('\\', '', $class);
+                if (isset($definition['classes']['model'])) {
+                    $class = $definition['classes']['pimcore_class_name'];
```
Comment (BlackbitDevs) [inline src/CoreShop/Bundle/ResourceBundle/Controller/ResourceSettingsController.php:63]: Shouldn't this be `$definition['classes']['model']`?

Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/ResourceBundle/Controller/ResourceSettingsController.php:63]: good point, will check again

**Comment by dpfaffenbauer** [comment]
@BlackbitDevs fixed it

**Pull request merged**